src/gatling/simulations/scenarios/Citizen_PRL_FL401_Respondent.scala

```diff
@@ -0,0 +1,451 @@
+
+	.group("PRL_FL401Respondent_130_SubmitPlaceOfBirthContinue") {
+	  exec(http("PRL_FL401Respondent_130_005_SubmitPlaceOfBirthContinue")
+	  .post(prlURL + "/respondent/confirm-contact-details/personaldetails")
+	  .headers(Headers.navigationHeader)
+      .header("accept", "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9")
+	  .formParam("_csrf", "#{csrf}")
+	  .formParam("citizenUserFirstNames", "#{citizenFirstName}")  
+	  .formParam("citizenUserLastNames", "#{citizenLastName}")
+	  .formParam("citizenUserAdditionalName", "")
+	  .formParam("citizenUserDateOfBirth-day", "#{citizenDOBDay}")
+	  .formParam("citizenUserDateOfBirth-month", "#{citizenDOBMonth}")
+	  .formParam("citizenUserDateOfBirth-year", "#{citizenDOBYear}")
+	  .formParam("citizenUserPlaceOfBirth", "#{PRLRandomString} City")
+	  .formParam("saveAndContinue", "true")
+      .check(CsrfCheck.save)
+	  .check(substring("Check your details")))
+	}
+
+	// Redirect to
+	//.exec(http("PRL_FL401Respondent_140_SubmitPlaceOfBirthCheckAnswers")
+	//  .get(prlURL + "/respondent/confirm-contact-details/checkanswers")
+	//  .headers(Headers.navigationHeader)
+    //  .header("accept", "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9")
+    //  .check(CsrfCheck.save))
+
+	.pause(MinThinkTime, MaxThinkTime)
+
+  //========================================================================================
+  // Save & Continue
+  //========================================================================================   
+
+	.group("PRL_FL401Respondent_140_Save&Continue") {
+	  exec(http("PRL_FL401Respondent_140_005_Save&Continue")
+	  .post(prlURL + "/respondent/confirm-contact-details/checkanswers")
+	  .headers(Headers.navigationHeader)
+      .header("accept", "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9")
+	  .formParam("_csrf", "#{csrf}")
+	  .formParam("citizenUserPlaceOfBirth", "#{PRLRandomString} City")
+	  .formParam("saveAndContinue", "true"))
+	}
+
+	.pause(MinThinkTime, MaxThinkTime)
+
+	// Redirect to
+	//.exec(http("PRL_FL401Respondent_150_Save&ContinueGetTasklist")
+	//  .get(prlURL + "/task-list/respondent")
+	//  .headers(Headers.navigationHeader)
+    //  .header("accept", "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9")
+    //  .check(CsrfCheck.save))
+
+
+  //========================================================================================
+  // Select Contact Preferences --- ** Not currently used within FL401 Respondent Journey  **
+  //========================================================================================   
+
+  val ContactPreferences = 
```

Comment:
There's 4 identical calls here? And no transaction numbers?

Reply:
Not currently used within FL401 Respondent Journey  (will be added in once functionality is in place)